api/services/MachineService.js

```diff
@@ -505,12 +503,16 @@ function updateMachinesPool() {
             let machineCreated = _.find(machinesCount.rows, (m) => m.image === image.id) || {count: 0};
             let machineToCreate = 0;
             let machineToDestroy = 0;
+            let machinePoolSize = config.machinePoolSize;
+            if (image.poolSize === null) {
+              machinePoolSize = config.machinePoolSize;
```

Comment:
This if is useless, But i think the first assignation should be: machinePoolSize = image.poolSize ?